tables/sr-cyr-chardefs.cti

```diff
@@ -0,0 +1,220 @@
+#
+#  Copyright (C) 2011 by Peter Engström <peter.engstrom@indexbraille.com>
+#  Copyright (C) 2011 by Monk Jeremiah, Manastir Decani <visokidecani@gmail.com>
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 3 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+#
+# -----------------------------------------------------------------------------
+#
+# 	SERBIAN
+#	=======
+# 	Authors:	Peter Engström, Index Braille <peter.engstrom@indexbraille.com>
+#				Monk Jeremiah, Manastir Decani <visokidecani@gmail.com>
+#       Aleksandar Sredojevic <ferrumite666@gmail.com>
+#
+# 	Date: 		Note:
+#	2011-03-28	Initial release
+#	2011-04-04	Added more signs.
+#	2015-08-28	Additions and fixes by Zvonimir Stanecic
+# 2024-03-20 Add a cyrillic back translation table
+#
+#---------------------------------------------------------------------------------------
+
+space \x00a0 a
+include spaces.uti
+
+#--------------------- ALL CHARS DEFINITIONS -------------------------------------------
```

Comment:
The way the tables are structured right now, there is a lot of duplication. I think we can eliminate pretty much all this duplication, by having two or more sub-tables that can be included by the sr-g1.ctb and sr-cyr-g1.ctb tables in the right order to get the desired effect.

Reply:
Pretty much done. Only duplication there still is are the notes in `sr-g1.ctb` and `sr-Cyrl-g1.ctb`, and the definitions of lj nj and dž because of noback.